### ISSUE TITLE
Trackpad Issues

### ISSUE THREAD

**Issue by programmatio** [mrphilipandrews/jquery.smoothwheel#21]
Doesn't play well with laptop trackpads or graphic tablets

**Comment by Kravimir** [comment]
I'm seeing this issue as well.

**Comment by harryparkdotio** [comment]
No, it definitely does not. The most notable issue is the fact that two finger pinch to zoom causes scrolling up and down the page.
